cfme/tests/configure/test_user_vm_list.py

```diff
@@ -0,0 +1,76 @@
+import cfme.configure.access_control as ac
```

Comment:
@rananda lets move this under the infrastructure folder and rename to test_vm_ownership.py